### DISCUSSION TITLE
mixcr exportClonesOverlap

### DISCUSSION THREAD

**Discussion by jum1996** [milaboratory/mixcr#1737] [Q&A]
I used mixcr exportClonesOverlap on .clns files that have been processed with mixcr findAlleles, and I got this following error:
 
 
`mixcr exportClonesOverlap --criteria "CDR3|AA|V|J" -isotype /home/jumana/Miami_aging/mixcr_miami_output2/allele/*.allelic.clns overlapTable.tsv
 `

```
[App version: 4.6.0-240-develop; built=Tue Jul 30 14:29:46 PDT 2024; rev=4adb74a18d; lib=repseqio.v5.0
Libraries have different checksums for library repseqio.v4.0_with_found_alleles and species null (taxonId=9606)

```
The catch is that when I ran exportClonesOverlap on .clns files that have been generated from mixcr analyze it works fine and I get a LARGE .tsv file back. I was wondering what the reason is this is happening. Is it a matter of compatibility? I'm aware that new reference libraries  are generated and clonotypes are re-aligned after findAlleles, but I was thinking that it would still work?

**Comment by mizraelson** [comment]
The development version has a slightly different file format, which is why some functions will only work if the data was initially analyzed with the same version. We try to maintain backward compatibility, but in some cases, it is not possible. When you analyze the data starting from FASTQ files with the same MiXCR version, do you still see this error?
